GameData/ROCapsules/PartConfigs/Orion/RP_Support_Orion.cfg

```diff
@@ -2,7 +2,7 @@
 //	Orion RealPlume Support
 //	============================================================================
 
-@PART[ROC-OrionESM]:NEEDS[RealPlume]:BEFORE[RealPlume]
+@PART[ROC-OrionESM]:BEFORE[RealPlume]:NEEDS[SmokeScreen]
```

Comment:
Note the previous version was redundant/over-specified.  BEFORE implies NEEDS.  This can just be `@PART[ROC-OrionESM]:BEFORE[RealPlume]` if we want to be consistent.